girder/girder_large_image/rest/large_image_resource.py

```diff
@@ -445,6 +447,72 @@ def _deleteCachedImages(self, spec, associatedImages=False, imageKey=None):
                 removed += 1
         return removed
 
+    @access.user(scope=TokenScope.DATA_WRITE)
+    @autoDescribeRoute(
+        Description('Create new large images for all items within a folder.')
+        .notes('Does not work for items with multiple files and skips over items with '
+               'existing or unfinished large images.')
+        .modelParam('id', 'The ID of the folder.', model=Folder, level=AccessType.WRITE,
+                    required=True)
+        .param('createJobs', 'Whether job(s) should be created for each large image.',
+               required=False, default=False, dataType='boolean')
+        .param('localJobs', 'Whether the job(s) created should be local.', required=False,
+               default=False, dataType='boolean')
+        .param('recurse', 'Whether child folders should be recursed.', required=False,
+               default=False, dataType='boolean')
+        .errorResponse('ID was invalid.')
+        .errorResponse('Write access was denied for the folder.', 403),
+    )
+    def createLargeImages(self, folder, params):
+        user = self.getCurrentUser()
+        createJobs = params.get('createJobs')
+        if createJobs:
+            createJobs = 'always' or True
```

Comment:
This seems to be a fragment from the item tiles endpoint with the force flag.  If this is a function renaming from the force flag, then it should go back to being called force for consistency and then internally, this goes back to `createJob='always' if self.boolParam('force', params, default=False) else True`.  The third state (don't use a job no matter what) would require the endpoint's parameter to accept three states (functionally these are `'always'`, never (False), and if-needed (True)).

Reply:
In order for the parameter to accept the three states, do you think it's best to make a string parameter and change createJobs accordingly? Should the flag still be called force?